test/message/promise_fast_handled_reject.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const p1 = new Promise((res, rej) => {
+  consol.log('One'); // eslint-disable-line no-undef
+});
+
+const p2 = new Promise((res, rej) => { // eslint-disable-line no-unused-vars
+  consol.log('Two'); // eslint-disable-line no-undef
+});
+
+const p3 = new Promise((res, rej) => {
+  consol.log('Three'); // eslint-disable-line no-undef
+});
+
+new Promise((res, rej) => {
+  setTimeout(common.mustCall(() => {
+    p1.catch(() => {});
+    p3.catch(() => {});
+  }), 1);
+});
+
+process.on('rejectionHandled', () => {}); // Ignore
```

Comment:
The default behavior for `rejectionHandled` is ignore.

Reply:
It will print a warning otherwise and I wanted to have the output as clean as possible.